### ISSUE TITLE
When combat-tagged by an Ender Pearl, players can still build in claims. (WeGank) 

### ISSUE THREAD

**Issue by CreatingWithKass** [PirateCraft/dev#127]
**Describe the bug** *(A clear and concise description of what the bug is)*:
Usually when a layer is combat-tagged, they cannot build in claims. However, when tagged by an Ender Pearl, they can still build. 

**To Reproduce** *(Steps to reproduce the behavior)*:
Combat tag a player with an Ender Pearl, and then have them try to build in a claim. It should allow them.

**Time and Date** *(If applicable, add the time (in GMT+1) and date (DD.MM.YYYY) of the issue)*:
20/03/21 

**Screenshots/Videos (Delete if not needed)** *(If applicable, add screenshots/videos to help explain your problem)*:
https://youtu.be/pd8NA_bJiKc

**Comment by TomLewis** [comment]
Snowball work the same?

**Comment by CreatingWithKass** [comment]
> Snowball work the same?

Yes, players can build in claims when combat tagged by a snowball as well.
